### ISSUE TITLE
jspm install --link fails in v0.16.0-beta.6

### ISSUE THREAD

**Issue by speigg** [jspm/jspm-cli#1007]
Using jspm v0.16.0-beta.6,  `jspm install -l <package>` and `jspm install --link <package>` have different behaviors. The `-l` option links fine, but the `--link` option is failing:

```
jspm install --link registry:name@version 

err  No linked versions found for jspm:--

warn Installation changes not saved.
```

**Comment by speigg** [comment]
Actually the options parsing in general is really buggy. Perhaps a rather than re-inventing the wheel here, it would be good to use https://github.com/substack/minimist or https://github.com/bcoe/yargs for doing the command line parsing.

**Comment by klausbayrhammer** [comment]
Thanks for the workaround with `jspm i -l ...` instead of `jspm i --link ...`

**Comment by guybedford** [comment]
This is fixed in https://github.com/jspm/jspm-cli/commit/11ad880a90cb0f2cc2477bd8891684ebee7dd8e8.

**Comment by guybedford** [comment]
Released in jspm@0.16.0-beta.7.
